frontend/src/features/verifiable-fields/components/VerificationBox/VerificationBox.tsx

```diff
@@ -20,6 +26,7 @@ type VfnFieldValues = {
 
 export interface VerificationBoxProps {
   fieldType: VerifiableFieldType
+  otpPrefix: string | null
```

Comment:
`otpPrefix?: string` would mean that the component allows for omission of `otpPrefix`

```jsx
// valid
<VerificationBox fieldType={type} otpPrefix={""} />

// also valid
<VerificationBox fieldType={type} />
```

But if we do `otpPrefix: string | null`, the component is required to handle and provide the `otpPrefix` props


```jsx
// valid
<VerificationBox fieldType={type} otpPrefix={""} />

// invalid
<VerificationBox fieldType={type} />

// now it is valid
<VerificationBox fieldType={type} otpPrefix={null} />
```

From the design standpoint, I was making the assumption that we want the `otpPrefix` to available rather than omitted.

Reply:
Ah, good point. In this case, is there a reason why we want to keep the null option instead of it being strictly a `string` then?

Reply:
Right. `null` is rather redundant here since the parent component would have been required to handle this case. Will remove.